numpy/core/_add_newdocs.py

```diff
@@ -3243,15 +3243,13 @@
         below. `new_order` codes can be any of:
 
         * 'S' - swap dtype from current to opposite endian
-        * {'<', 'L'} - little endian
-        * {'>', 'B'} - big endian
-        * {'=', 'N'} - native order
+        * '<' - little endian
+        * '>' - big endian
+        * '=' - native order
```

Comment:
Do we want to explicitly indicate we accept the `sys.byteorder` names?
```suggestion
        * {'<', 'little'} - little endian
        * {'>', 'big'} - big endian
        * '=' - native order, equivalent to `sys.byteorder`
```

Reply:
not sure, sys.byteorder means only 'big' or 'little' and this list is a superset of that. I dont think it has any negative effect though.